server/src/routes/opportunities/index.ts

```diff
@@ -200,31 +245,45 @@ const saveCVData = (_: ILogger) => async (ctx: Router.RouterContext) => {
     ...data,
   });
 
-  setResponse(ctx, OK, result);
+  const { id, expires, githubId: omittedGithubId, isHidden, ...dataToSend } = result;
+
+  setResponse(ctx, OK, dataToSend);
 };
 
-export const createCV = (_: ILogger) => async (ctx: Router.RouterContext) => {
-  const { githubId } = ctx.state!.user as IUserSession;
+const getOpportunitiesConsent = (_: ILogger) => async (ctx: Router.RouterContext) => {
+  const { githubId } = ctx.request.query;
+
+  const userRepository = getRepository(User);
+  const user = await userRepository.findOne({ where: { githubId } });
+
+  if (user === undefined) {
+    setResponse(ctx, NOT_FOUND);
+    return;
+  }
 
+  setResponse(ctx, OK, user.opportunitiesConsent);
+};
+
+const createCV = async (githubId: string) => {
   const cvRepository = getRepository(CV);
 
   const cv = await cvRepository.findOne({ where: { githubId } });
 
-  if (cv != undefined) {
-    setResponse(ctx, CONFLICT);
-    return;
+  if (cv !== undefined) {
+    return 'ALREADY_EXISTS';
```

Comment:
Can you create some type with all this stuff 'ALREADY_EXISTS', 'CREATED', etc.?
like:
`type CvStatus = 'ALREADY_EXISTS' | 'CREATED'`

Reply:
enum created, done